src/joint.dia.element.js

```diff
@@ -694,8 +694,6 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
     pointerdown: function(evt, x, y) {
 
-        this.model.trigger('batch:start');
-
         // target is a valid magnet start linking
         if (evt.target.getAttribute('magnet') && this.paper.options.validateMagnet.call(this.paper, this, evt.target)) {
 
```

Comment:
Here we are missing a batch now. A new link is added to the graph with no previous batch start resulting that one user action (from `pointerdown` to `pointerup`) it's not entirely wrapped in a batch.
I'm not sure what are you using batches for, but one of the usage might be a history (undo/redo) implementation, where creating a link, dragging its arrowhead and then connecting it to an element should be an atomic action.